packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -450,12 +455,23 @@ export default class Dropdown extends React.Component<DropdownProps, State> {
             : 0;
 
         let focusCounter = 0;
+        let itemRole;
+
+        switch (this.props.role) {
+            case "listbox":
+                itemRole = "option";
+                break;
+            case "menu":
+                itemRole = "menuitem";
+                break;
```

Comment:
There's a hybrid action menu in https://deploy-preview-357--wonder-blocks.netlify.com/#!/ActionMenu which includes menu items and options.  I wonder if it makes sense to embed the role in the ActionItem and OptionItem instead of basing it on the container's `role`.

Reply:
I checked the code at that's what we're doing.  I'm going to get rid of this.

Reply:
I chatted with @somewhatabstract on Slack and it sounds like what we have here is actually what we want so I'm going to leave it.  I just need to pass this down to `ActionItem` and `OptionItem`.